cloudbuild-lint-and-test.yaml

```diff
@@ -0,0 +1,34 @@
+steps:
+  # Runs Megalinter on the codebase
+  - name: docker
+    id: megalint_proxy_service
+    entrypoint: "sh"
+    args:
+      - "-c"
+      - |
+        echo "*************************************************************"
+        echo "* Running Megalinter on the codebase...                     *"
+        echo "*************************************************************"
+        docker run --platform linux/amd64 --rm \
```

Comment:
I think this build `step` in Cloud Run is run inside a docker a container already, so this will run a docker container inside another docker container so not sure if this looks quite right?

Reply:
I does look odd, will need to break down what this is doing exactly

Reply:
Would this be done as a separate piece of work?